composer.lock

```diff
@@ -1022,12 +1022,12 @@
             "source": {
                 "type": "git",
                 "url": "https://github.com/WordPress/WordPress-Coding-Standards.git",
-                "reference": "73f3d69b3a200935b80f5fd265bfbee8d642d082"
+                "reference": "7b39722c38cbf2fe64c55be7154ee6f1807d304c"
```

Comment:
This happened after running `tools/composer-update-monorepo.sh --root-reqs "."`, which was recommended as a fix to a failing check. I'm not sure if it's an issue to update this reference.